test/api_spec.js

```diff
@@ -77,6 +77,65 @@ describe("api helper", () => {
     });
   });
 
+  describe("updateReportSettingsInPortal", () => {
+    // need a fake offering url param that includes /offerings/
+    const fakeOfferingUrl = "https://portal.com/offerings/123";
+    const okResponse = "OK";
+    const fakeSettings = {something: "important"};
+
+    beforeEach(() => {
+    });
+
+    describe("when offering and token URL param is present", () => {
```

Comment:
Would it be a little bit more clear to say something like "when both offering and token URL params are present"?

Reply:
Yup, thanks.